ntoskrnl/ex/handle.c

```diff
@@ -67,12 +69,14 @@ ExpLookupHandleTableEntry(IN PHANDLE_TABLE HandleTable,
 
             /* Get the mid level pointer array */
             PointerArray = PointerArray[Handle.HighIndex];
+            ASSERT(PointerArray == NULL);
```

Comment:
Did you mean !=?

Reply:
Yea, probably. Will look into that later